pages/report.vue

```diff
@@ -0,0 +1,303 @@
+<template>
+  <div class="page">
+    <Card>
+      <div class="content">
+        <div>
+          <h1 class="card-title-adjustments">Submit a Report</h1>
+          <div class="">
+            <p>
+              We're committed to maintaining a safe and trustworthy platform. If you encounter
+              content that violates our
+              <nuxt-link class="text-link" to="/legal/terms">Terms of Service</nuxt-link> or our
+              <nuxt-link class="text-link" to="/legal/rules">Rules</nuxt-link>, please report it to
+              us here.
+            </p>
+            <p>
+              This form is intended exclusively for reporting abuse or harmful content to Modrinth
+              staff. For bugs related to specific projects, please use the project's designated
+              Issues link or Discord channel.
+            </p>
+            <p>
+              Your privacy is important to us; rest assured that your identifying information will
+              be kept confidential.
+            </p>
+          </div>
+        </div>
+        <div>
+          <div class="report-info-section">
+            <div class="report-info-item">
+              <label for="report-item">Item Type to Report</label>
```

Comment:
Never use title case for anything that isn't explicitly a title.
```suggestion
              <label for="report-item">Item type to report</label>
```